### ISSUE TITLE
Check for wireguard org state continues to pass after removing all wireguard connections

### ISSUE THREAD

**Issue by marcus-crane** [superfly/flyctl#446]
**Describe the bug**
One possible state for the user to get into (I have never gotten myself into this state accidentally *cough*) is one where they've deleted all of their wireguard connections. Most functionality continues to work as `~/.fly/config.yml` contains a valid access token that can be used to query the graphql endpoint.

However, if a user then tries to run `fly ssh console`, well, here's an image. Starting a new session shouldn't matter so much but the top session is the first one and the bottom is the second/newer.

![Screen Shot 2021-07-03 at 3 47 09 PM](https://user-images.githubusercontent.com/14816406/124342291-00c1ee00-dc17-11eb-98f3-1bcb9de02484.png)

What appears to be going on here is that the call to [StateForOrg](https://github.com/superfly/flyctl/blob/539d4cff129a083c8482ad3c1a4b10b290faae45/internal/wireguard/wg.go#L51) sees that there is a syntactically valid configuration and so, it just simply returns the existing config. Of course, what it should probably do is verify that the connection still exists.

Deleting `~/.fly/config.yml` fixes this issue pretty easily (as the next call to StateForOrg regenerates the `wire_guard_state` block) but that also nukes your access token and forces you to log back in so just deleting the wireguard state block is quicker.

I don't imagine this is related but I've also noticed the initial connection after creating a new peer is a little flaky?

```
fly ssh console -a gunslinger
Creating WireGuard peer "interactive-amaterasu-marcus-utf9k-net-601" in region "ord" for organization personal
Connecting to gunslinger.internal...⢿ Error connect to SSH server: dial: lookup gunslinger.internal. on fdaa:0:2d9f::3: read udp [fdaa:0:2d9f:a7b:bea:0:a:200]:16722: i/o timeout
 fly ssh console -a gunslinger
Connecting to gunslinger.internal... complete
```

It usually takes me two or three connections before the connection establishes so I imagine there's some sync delay between talking with graphql and it propogating whereever it needs to.

**Add `fly.toml`**
Doesn't apply as I'm running with the `-a` flag

**Command output:**
Maybe a bit redundant after the explainer 😛

**Comment by marcus-crane** [comment]
Submitted an attempt at fixing this bug

**Comment by michaeldwan** [comment]
Fixed in #483
